src/App.js

```diff
@@ -1,5 +1,99 @@
+import { Console, Random } from "@woowacourse/mission-utils";
+import Car from "./Car.js";
+import { validateCarNameList, validateTryCount } from "./validation.js";
+import {
+  CAR_NAME_INPUT_MESSAGE,
+  TRY_COUNT_INPUT_MESSAGE,
+} from "./constants/inputMessage.js";
+import {
+  EMPTY_MESSAGE,
+  EXECUTE_RESULT_MESSAGE,
+  WINNER_MESSAGE,
+} from "./constants/outputMessage.js";
+
 class App {
-  async run() {}
+  #tryCount;
+  #cars;
+
+  constructor() {
+    this.#tryCount = 0;
+    this.#cars = [];
+  }
+
+  async run() {
+    let carNameList = await this.readCarName();
+    validateCarNameList(carNameList);
+    this.generateCars(carNameList);
+
+    const tryCount = await this.readTryCount();
+    validateTryCount(Number(tryCount));
+    this.#tryCount = Number(tryCount);
```

Comment:
유저가 입력한 시도횟수와 유효성 검사를 마친 후 private 필드로 저장된 값을
구분할 수 있게 변수명을 바꾸면 실수를 줄일 수 있을 것 같아요! 😊

```suggestion
    const inputTryCount = await this.readTryCount();
    validateTryCount(Number(inputTryCount));
    this.#tryCount = Number(inputTryCount);
```